CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # Azure Kubernetes Service Changelog
 
+
+## Release 2024-04-11
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+- [Dual-stack networking (IPv4/IPv6) with CNI Overlay](https://learn.microsoft.com/en-us/azure/aks/azure-cni-overlay?tabs=kubectl#dual-stack-networking) is now generally available (GA) for Linux node pools and does not need the Azure Feature Flag registered on the subscription.
+
+### Release Notes
+
+* Features:
+
+
+* Preview Features:
+
+
+
+
+* Behavioral Changes:
+-   Clusters running Kubernetes 1.29 or later will have kubernetes.azure.com/managedby=aks  label to tigera-operator deployment in Calico clusters 
+- This introduces the constraint template validation behavior change called out in November's release notes
+[2023-11-28](https://github.com/Azure/AKS/releases/tag/2023-11-28) . It also improves cleanup of the addon, as called out in  Issue [#3541](https://github.com/Azure/AKS/issues/3541) , and patches a CVE in the addon.
+- Adds [App routing](https://learn.microsoft.com/en-us/azure/aks/app-routing?tabs=default%2Cdeploy-app-default) to excluded namespaces for policy when the addon is enabled
+- Toggle OSI clusters with 3.1.17 for autoscale
```

Comment:
may not be relevant for user facing release notes. can we check internally on this with author of the relevant PR

Reply:
Asked the PR owners